### PR TITLE
[#157916427] Installazione dipendenza di bootstrap-italia. Inizio a r…

### PR DESCRIPTION
**Pull request by vito80ba**
[#157916427] Installazione dipendenza di bootstrap-italia. Inizio a rimuovere la semantica delle classi suitcss in favore di bootstrap4

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -126,3 +126,9 @@ page_gen:
     template: api-details
     dir: en/api
     filter_condition: "record['language'].include? 'en'"
+
+sass:
+    load_paths:
+      - _sass
+      - node_modules
+    sass_dir: /
```
Comment (lussoluca) [inline _config.yml]: always leave an empty line at the end of all files

File: assets/main.scss [hunk 2]
```diff
@@ -2,8 +2,26 @@
...
+      line-height: 1.555;
+    }
+}
+
+// STILE DEVELOPERS
+.Header-logo {
+    img {
+        width: 65px;;
+    }
+}
```
Comment (lussoluca) [inline assets/main.scss]: always leave an empty line at the end of all files

File: _includes/head.html [hunk 3]
```diff
@@ -35,12 +34,8 @@
     })();
   </script>
 
-  <!-- include html5shim per Explorer 8 -->
-  <script src="{{ '/assets/toolkit/vendor/modernizr.js' | relative_url }}"></script>
```
Comment (lussoluca) [inline _includes/head.html]: by removing this we drop the support for IE8?

File: package.json [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+        "bootstrap-italia": "*",
+        "jquery": "^3.3.1",
+        "pickadate": "^3.5.6",
+        "popper.js": "^1.14.3"
+    },
+    "repository": {
+        "type": "git",
+        "url": "git://github.com/italia/developers.italia.it"
+    },
+    "license": "See license in LICENSE file"
```
Comment (lussoluca) [inline package.json]: `npm install` complains that this is not a valid license

File: _includes/head.html (hunk 3 above)
Comment (vito80ba) [reply _includes/head.html]: Bootstrap4 is not ie8 compliant

File: package.json (hunk 4 above)
Comment (vito80ba) [reply package.json]: Ask to @ruphy

File: _includes/head.html (hunk 3 above)
Comment (lussoluca) [reply _includes/head.html]: ok, not a problem for me, we just need to agree that the new version of the website will not be compatibile with IE8. @rasky is this OK for you?

**Pull request merged**